data/src/main/java/com/microsoft/azure/kusto/data/http/HttpClientProperties.java

```diff
@@ -118,9 +129,9 @@ public static class HttpClientPropertiesBuilder {
         private Integer maxKeepAliveTime = 120;
         private Integer maxConnectionsTotal = 40;
         private Integer maxConnectionsPerRoute = 40;
-        private HttpHost proxy = null;
```

Comment:
I would avoid using the builder in your comment. The word Netty should be considered a dirty word because it is a specific implementation and locks the customer into using the Netty client and locks us into using the Netty client as well. 

If you wanted to use OkHttp instead of Netty, the code in the comment above would have to be replaced first.

Inversion of dependencies says to depend on interfaces instead of implementations, in this case everything we have done thus far in this branch uses the azure-http-client interface and does not lock you in to a specific implementation.

Reply:
I've opened an issue over on azure-core for this.
https://github.com/Azure/azure-sdk-for-java/issues/42083